src/pages/transactions/[id].js

```diff
@@ -1,42 +1,42 @@
 import { useRouter } from "next/router";
 import Link from "next/link";
 import Modal from "@/src/components/Modal";
-import ChevronLeft from "@/src/assets/icons/chevron_left.svg";
-import DeleteBin from "@/src/assets/icons/delete.svg";
+import { useState } from "react";
 
 export default function TransactionDetails({
-                                             initialData,
+                                             transactions,
                                              onDelete,
-                                             mode,
-                                             onMode,
                                            }) {
   const router = useRouter();
   const { id } = router.query;
+  const [modal, setModal] = useState(false);
 
   // Find transaction by its ID
-  const transaction = initialData.find((transaction) => transaction.id === id);
+  const transaction = transactions.find((transaction) => transaction.id === id);
 
   if (!transaction) {
     return <p>Transaction not found</p>;
   }
 
+  const displayAmount = `${transaction.transactionType === "Expense" && "-"}${transaction.amount.toFixed(2)}`;
+
   return (
     <>
       <div>
```

Comment:
If you want to group the content within a ```<div>```maybe a ```<section>```is a good alternative.

Reply:
Will be changed in the styling ticket